virtualpdu/tests/unit/drivers/test_libvirt_driver.py

```diff
@@ -19,7 +19,8 @@
 from virtualpdu.drivers import libvirt_driver
 from virtualpdu.tests import base
 
-DOMAIN_ALREADY_RUNNING = "internal error: Domain '%s' is already running"
+DOMAIN_ALREADY_RUNNING = "internal error: Domain {} is already running"
+DOMAIN_NOT_RUNNING = "internal error: Domain {} is not running"
```

Comment:
Fboulianes bug report mentioned 'Domain is not running', without the domain name. I suggest we mimic this here